config/Config-devel.in

```diff
@@ -17,6 +17,41 @@ menuconfig DEVEL
 		  Store built firmware images and filesystem images in this directory.
 		  If not set, uses './bin/$(BOARD)'
 
+	menuconfig DOWNLOAD_TOOL_USE_CUSTOM
+		bool
```

Comment:
I would name it `DOWNLOAD_TOOL_CUSTOM` and make it string, so keeping it simple and future proof. That way it doesn't matter if you use aria2c, super-curl3, spider-fu8 or monkey-get3000.

Reply:
sooo dropping the selection and just make the user insert a string? download.pl still needs to have support for it, that is why i wanted a selection so the user know what is supported.

Reply:
> download.pl still needs to have support for it

Reply:
@ynezz ok tried to do my best to find a ""future proof" and keep support with the 3 tools we support. What do you think?